src/sentry/data/samples/cocoa.json

```diff
@@ -5,17 +5,20 @@
       {
         "value": "Attempted to dereference null pointer.\nOriginated at or in a subcall of ViewController.causeCrash(Any) -> ()",
         "mechanism": {
-          "posix_signal": {
```

Comment:
Can we still keep the tests with the old mechanism? 
I guess it makes sense since it will take a while until we actually see new mechanism appear.

Reply:
We do have them still. When you search the code base for `posix_signal` you'll find some unit tests. Or would you prefer full acceptance tests, too?